### PR TITLE
Add linkdrop redirect for existing users

### PR DESCRIPTION
**Pull request by think-in-universe**
### Description

Resolves: #1827

Similar to #1641 which is designed specific for new users, add url redirect support after an existing user claimed the linkdroip.

### CODE REVIEW

**Comment by think-in-universe** [comment]
Before we merge the PR, we require more security review as mentioned here by @stefanopepe https://github.com/near/near-wallet/issues/1827#issuecomment-863440319

**Comment by think-in-universe** [comment]
Any ideas about the CI error? does it fail because I didn't submit the branch to https://github.com/near/near-wallet repo directly?

**Comment by MaximusHaximus** [comment]
> Any ideas about the CI error? does it fail because I didn't submit the branch to https://github.com/near/near-wallet repo directly?

Yes, I think that is the problem; it looks like we are initializing Sentry during the build process when we don't need to and really shouldn't be for branch PR builds.  I'll take a look into this and figure out the right way to resolve it :+1:, but I don't think it's a blocker.

File: src/utils/wallet.js
Comment (MaximusHaximus) [inline src/utils/wallet.js:443]: I know that we're already dispatching using `store` in this file, so this wasn't obvious, but we're trying to extricate Redux awareness from `wallet.js`.  Instead of dispatching here, can you please enhance the `CLAIM_LINKDROP_TO_ACCOUNT` action in `actions/account.js` to call this wallet method, then dispatch this action instead of having this wallet method dispatch the action?

**Review by MaximusHaximus — CHANGES_REQUESTED** [review]
Keen to get this merged this sprint, but just one change requested around the usage of `store` in wallet.js, which we're trying to move away from :)

File: src/utils/wallet.js
Comment (think-in-universe) [reply src/utils/wallet.js]: sure. I have updated the action `CLAIM_LINKDROP_TO_ACCOUNT `

Comment (MaximusHaximus) [reply src/utils/wallet.js]: :+1: thanks so much

**Review by MaximusHaximus — APPROVED** [review]
:+1: :shipit:

**Pull request merged**
